packages/ui/src/components/SliderSteps/SliderSteps.tsx

```diff
@@ -2,36 +2,40 @@ import React from 'react';
 import { Center } from '@chakra-ui/react';
 
 import { OutlinedArrowLeftIcon, OutlinedArrowRightIcon } from '../../icons';
+import { IconButton } from '../IconButton';
 import { SliderStep } from './SliderStep';
 
 export interface SliderStepsProps {
   count: number;
-  selectedIdx: number;
-  onChangeIdx: (newIdx: number) => void;
+  selectedIndex: number;
+  onChange: (newIdx: number) => void;
 }
 
-export const SliderSteps = ({ count, selectedIdx, onChangeIdx }: SliderStepsProps) => {
+const arrowStyles = {
+  variant: 'ghost',
+  size: 'sm',
+} as const;
+
+export const SliderSteps = ({ count, selectedIndex, onChange }: SliderStepsProps) => {
   const lastIdx = count - 1;
-  const arrowStyles = {
-    w: '32px',
-    h: '32px',
-    p: '6px',
-    borderRadius: '6px',
-    color: 'gray.800',
-  };
+  const goForward = () => onChange(selectedIndex !== lastIdx ? selectedIndex + 1 : 0);
+  const goBack = () => onChange(selectedIndex !== 0 ? selectedIndex - 1 : lastIdx);
+  const goToSelected = (index: number) => onChange(index);
```

Comment:
💭 You can check if the index is the current one and in such case don't call the `onChange` function